src/assets/js/mailoptin.min.js

```diff
@@ -1,2 +1,3309 @@
\ No newline at end of file
+(function ($, require$$0, mailoptin_globals) {
```

Comment:
Remove your minfied mailoptin.js too @ShoboySnr

Reply:
@ShoboySnr please cleanup the PR so i can review

Reply:
Okay I'll